### DISCUSSION TITLE
A new description of the package

### DISCUSSION THREAD

**Discussion by adw96** [statdivlab/rigr#88] [General]
Hi @statdivlab/rigr-ta-team and @jphughes9 -- I'm just modifying the "description" field in DESCRIPTION. I'm sure I've missed some key things

_A set of tools to streamline data analysis in R. Learning both R and introductory statistics at the same time can be challenging, and so we created `rigr` to facilitate common data analysis tasks and to enable learners to focus on statistical concepts. We provide easy-to-use interfaces for descriptive statistics, one- and two-sample inference, and regression analyses. Heteroskedasticity-robust ("sandwich") standard errors are included by default in all regression output, and multiple partial F-tests are easy to specify. A single regression function (`regress()`) can fit both linear and generalized linear models, allowing students to more easily make connections between different classes of models._ 

For reference, here is old description:

_A set of tools designed to facilitate easy adoption of R for students in introductory classes with little programming experience. Compiles output from existing routines together in an intuitive format, and adds functionality to existing functions. For instance, the regression function can perform linear models, generalized linear models, Cox models, or generalized estimating equations. The user can also specify multiple-partial F-tests to print out with the model coefficients. We also give many routines for descriptive statistics and plotting._

Thoughts/edits?

**Comment by jphughes9** [comment]
Amy

This sounds good to me. Should we announce the package at the faculty meeting tomorrow?

Jim
--------------
Jim Hughes
Professor of Biostatistics
University of Washington           Email: ***@***.******@***.***>
Mailstop 351617                           Phone: 206-616-2721
Seattle, WA  98195                        Fax:      206-616-2724
                                                          Cell:      206-498-2456


From: AmyW ***@***.***>
Sent: Wednesday, September 8, 2021 11:36 AM
To: statdivlab/rigr ***@***.***>
Cc: Jim Hughes ***@***.***>; Mention ***@***.***>
Subject: [statdivlab/rigr] A new description of the package (#88)


Hi @statdivlab/rigr-ta-team<https://github.com/orgs/statdivlab/teams/rigr-ta-team> and @jphughes9<https://github.com/jphughes9> -- I'm just modifying the "description" field in DESCRIPTION. I'm sure I've missed some key things

A set of tools to streamline data analysis in R. Learning both R and introductory statistics at the same time can be challenging, and so we created rigr to facilitate common data analysis tasks and to enable learners to focus on statistical concepts. We provide easy-to-use interfaces for descriptive statistics, one- and two-sample inference, and regression analyses. Heteroskedasticity-robust ("sandwich") standard errors are included by default in all regression output, and multiple partial F-tests are easy to specify. A single regression function (regress()) can fit both linear and generalized linear models, allowing students to more easily make connections between different classes of models.

For reference, here is old description:

A set of tools designed to facilitate easy adoption of R for students in introductory classes with little programming experience. Compiles output from existing routines together in an intuitive format, and adds functionality to existing functions. For instance, the regression function can perform linear models, generalized linear models, Cox models, or generalized estimating equations. The user can also specify multiple-partial F-tests to print out with the model coefficients. We also give many routines for descriptive statistics and plotting.

Thoughts/edits?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/statdivlab/rigr/discussions/88>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AFPGUCS6Q3IAEGM2FRAJSXDUA6UJFANCNFSM5DVOFLMA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by adw96** [comment]
Note to self: perhaps include mention of tests of contrasts via lincom?

**Comment by taylorokonek** [comment]
The description sounds good to me! No huge opinion on whether or not lincom should be mentioned

**Comment by cwolock** [comment]
Looks good to me. 

Maybe one thing that could be added (but I don't feel strongly) is something about how we're trying to replicate what students learn in an intro class. I've been thinking about this a lot lately with `proptest` - the `stats` package version does some weird stuff that I think is probably opaque to a beginner, while our version does exactly the Wald test and confidence interval that is taught in an intro course.

**Reply by adw96** [reply]
Thanks, @cwolock ! Great point. I've added "'rigr' output includes key information while omitting unncessary details that can be confusing to beginners." in `4ff77c2`
